Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -6568,18 +6568,17 @@ static RefPtr<CSSValue> consumeCounterContent(CSSParserTokenRange args, bool cou
         separator = args.consumeIncludingWhitespace().value().toAtomString();
     }
 
-    auto listStyle = CSSValueDecimal;
+    RefPtr<CSSValue> listStyleType = CSSPrimitiveValue::create(CSSValueDecimal);
```

Comment:
Not sure this will be accurate when @counter-style is on.

Reply:
When it's on, predefined values are appropriately converted to ListStyleType with CounterStyle as type and an identifier.